packages/php-wasm/universal/src/lib/fs-helpers.ts

```diff
@@ -198,6 +198,21 @@ export class FSHelpers {
 		return FS.isFile(FS.lookupPath(path).node.mode);
 	}
 
+	static isSymlink(FS: Emscripten.RootFS, path: string): boolean {
+		if (!FSHelpers.fileExists(FS, path)) {
+			return false;
+		}
+
+		return FS.isLink(FS.lookupPath(path).node.mode);
```

Comment:
Add `?` to avoid blowups in case `node` doesn't not exist.

Reply:
Checked implementation and it seems we always return an object with `node` property.

However, `lookup` can throw.
What about wrapping it in a `try..catch` and return false if there is an error?

Reply:
I'd say that it should work as we already checked if the file exists. If this fails, I would think it is ok to throw the exception and let the calling code handle it. What do you think?

Reply:
> I'd say that it should work as we already checked if the file exists. If this fails, I would think it is ok to throw the exception and let the calling code handle it. What do you think?

Since that is the general pattern with FSHelpers, I think that is right.